app/scripts/lib/snap-keyring/types.ts

```diff
@@ -31,10 +32,12 @@ export type SnapKeyringBuilderAllowActions =
   | GetSubjectMetadata
   | AccountsControllerSetSelectedAccountAction
   | AccountsControllerGetAccountByAddressAction
-  | AccountsControllerSetAccountNameAction;
+  | AccountsControllerSetAccountNameAction
+  | HandleSnapRequest
+  | GetSnap;
 
 export type SnapKeyringBuilderMessenger = RestrictedControllerMessenger<
-  'SnapKeyringBuilder',
+  'SnapKeyring',
```

Comment:
We have to update this since the AccountsController uses this messenger's name now for the new keyring events.